src/build.py

```diff
@@ -293,8 +293,6 @@ def CopyLibraryToSysroot(library):
 def Archive(directory, print_content=False):
   """Create an archive file from directory."""
   # Use the format "native" to the platform
-  if not buildbot.IsBot():
-    return
```

Comment:
Did you mean to remove this?

Reply:
Yes, see the "Also" part of the comment.